Source/Particle.cpp

```diff
@@ -191,7 +191,7 @@ PeleC::defineParticles()
   if (SPRAY_FUEL_NUM > NUM_SPECIES) {
     Abort("Cannot have more spray fuel species than fluid species");
   }
-#ifdef PELEC_EOS_FUEGO
```

Comment:
Not familiar with this part of the code or if coupling sprays and SRK is meant to be supported at all, but if so it seems to me like SRK should be treated the same as Fuego rather than the same as GammaLaw here.

Reply:
good catch!